tests/tests_ut/az_ulib_ipc_ut/az_ulib_ipc_ut.c

```diff
@@ -2643,6 +2874,184 @@ static void az_ulib_ipc_get_table_succeed(void** state)
   /// cleanup
 }
 
+/* The az_ulib_ipc_split_method_name shall split the method full name returning device, package,
```

Comment:
shouldn't these tests be with the tests ending on line 937?

Reply:
No, those are the Precondition tests in the UT. They are inside of `AZ_NO_PRECONDITION_CHECKING`. After that we have the tests not related to the precondition.